src/pages/flags/index.page.tsx

```diff
@@ -0,0 +1,49 @@
+import { Badge } from 'components/Badge'
+import { JuiceSwitch } from 'components/JuiceSwitch'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+import { readNetwork } from 'constants/networks'
+import { useCallback, useState } from 'react'
+import { featureFlagEnabled, setFeatureFlag } from 'utils/featureFlags'
+import Head from 'next/head'
+
+export default function FlagsPage() {
+  const [, updateState] = useState({})
+  const forceUpdate = useCallback(() => updateState({}), [])
```

Comment:
i have sinned

Reply:
explain

Reply:
@wraeth-eth component doesnt re-render when localstorage is updated, so need to trigger a re-render somehow. Feels hacky but idk, probably fine.

Reply:
oh lol - I see. Sounds good. Maybe a better way, but not worth slaving over